raptor/playback/mitmproxy.py

```diff
@@ -0,0 +1,38 @@
+'''This helps loading mitmproxy's cert and change proxy settings for Firefox.'''
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from __future__ import absolute_import
+
+import os
+
+from mozlog import get_proxy_logger
+
+from .base import Playback
+
+here = os.path.dirname(os.path.realpath(__file__))
+tooltool_cache = os.path.join(here, 'tooltoolcache')
+
+LOG = get_proxy_logger(component='mitmproxy')
+
+
+class Mitmproxy(Playback):
+
+    def __init__(self, config):
+        self.config = config
+
+    def download(self):
+        LOG.info("todo: download mitmproxy release binary")
+        return
+
+    def setup(self):
```

Comment:
Is there a time you'd want to create a Playback class but not set it up right away? If not maybe the `download` and `setup` could just be called from `__init__` (and removed from the base class).

Reply:
Good point! If every Playback class will need to be downloaded+setup, then yes you should leave it in the base class. As a minor cleanup, you could even call them from the base class' init (and make sure Mitmproxy either invokes that from its own init, or doesn't have an init)

Reply:
I think I'd rather leave it up to the derived playback tool classes to decide that - never know, in the new tool we will switch too eventually it may need to be created but not setup at the same time